app/mailers/user_mailer.rb

```diff
@@ -83,10 +83,10 @@ def no_dump(target_address)
     mail(:subject => "openSNP.org: Sorry, there is no data to be dumped.", :to => target_address)
   end
 
-  def fitbit_dump(link,user)
+  def fitbit_dump(link, user_id)
     @link = link
-		@user = user
-    mail(:subject => "openSNP.org: The Fitbit-data you requested is now ready for download",:to => @user.email)
+		@user = User.find(user_id)
```

Comment:
Seems that was one of the cases where Atom screwed up in detecting on whether it should use tabs or spaces, as it usually does this automatically :-)